spec/models/waste_carriers_engine/order_item_spec.rb

```diff
@@ -36,12 +36,13 @@ module WasteCarriersEngine
       end
 
       it "should set the correct description" do
-        expect(order_item.description).to eq("Renewal of registration")
+        expect(order_item.description).to eq("renewal of registration")
```

Comment:
I am puzzled by this. Given that we have https://github.com/DEFRA/waste-carriers-engine/pull/733/files#diff-1aaae17839a245710574a863973dae8bR97, shouldn't this also be setting a description that starts with a capital letter?

Reply:
I moved the capitalising to the Order model as otherwise we ended up with some wonky capitalising in the joined-up description there. I figured it was better to make the individual OrderItems consistent and then capitalise them when we combine them.

Reply:
Ohhh I got it now! Make sense! :D